apps/nowcasting-app/components/types.d.ts

```diff
@@ -78,7 +78,12 @@ type GspAllForecastData = {
 type GspDeltaValue = {
   gspId: number;
   gspRegion: string;
+  gspInstalledCapacity: number;
   currentYield: number;
   forecast: number;
   delta: number;
+  deltaColor: string;
+  dataKey: string;
+  deltaPercentage: string;
+  deltaNormalized: string;
```

Comment:
Oh, looks like you have the colour/key/percent stuff in here already to be set in the data..?

Reply:
yeah, I did add those to the type so I could get the deltaPercentage and deltaNormalized but sometimes I feel pretty confused on how to use types. should look at this too when updating the formatChartData function.